lib/backburner.js

```diff
@@ -73,6 +79,48 @@ Backburner.prototype = {
     }
   },
 
+  _trigger: function(eventName /* ...args */) {
+    var callbacks = this._eventCallbacks[eventName];
+    if (callbacks) {
+      var len = arguments.length;
+      var args = new Array(len - 1);
+      for (var i = 1; i < len; i++) {
+        args[i - 1] = arguments[i]
+      }
+      for (var i = 0; i < callbacks.length; i++) {
+        callbacks[i].apply(null, args);
+      }
+    }
+  },
+
+  on: function(eventName, callback) {
+    var callbacks = this._eventCallbacks[eventName];
+    if (callbacks) {
+      callbacks.push(callback);
+    }
+  },
+
+  off: function(eventName, callback) {
+    if (eventName) {
```

Comment:
this should be based on arity, not truthyness, if someone accidentally passes `off(undefined)` they will pwn all subscribers.

I actually think we shouldn't bother supporting the batch `off` case, it seems problematic.